### PR TITLE
feat: `Product Info` set context values

### LINKED ISSUE #2043: Información de Producto completa
## Feature request

Se define que el **buscador de productos** será utilizado para poder buscar productos fácilmente, no para ver información general de productos como lo hace ahora ZK q en una misma ventana rinde para las dos necesidades diferentes.

Para estas necesidades de ver información general de productos, le daremos más funcionalidad a la forma **"Información de Productos"** que ahora está muy ligado al punto de venta pero en realidad no debería ser así, dejandolo libre para que toda organización pueda ver la información de sus productos desde este visor. Para esto crearé una nueva issue:

image

image

#### Describe alternatives you've considered

Información de Producto completa

### PR DESCRIPTION
**Pull request by elsiosanchez**
#### Screenshot or Gif

https://github.com/solop-develop/frontend-core/assets/45974454/b8b22012-8b98-471b-88ab-11d2a4db8872

### CODE REVIEW

File: src/components/ADempiere/Form/ProductSearch/index.vue [hunk 1]
```diff
@@ -341,8 +394,86 @@ export default defineComponent({
...
+
+    function close() {
+      closeTagView()
+    }
+
+    function setContextValue() {
+      store.commit('setProductSearchFieldQueryFilterByAttribute', {
+        containerUuid: uuidForm,
+        attributeKey: 'warehouse_id',
+        value: getContext({
```
Comment (EdwinBetanc0urt) [inline src/components/ADempiere/Form/ProductSearch/index.vue]: Please validate if context contains this value and value is `> 0` to mutate vuex state.

File: src/components/ADempiere/Form/ProductSearch/index.vue [hunk 2]
```diff
@@ -341,8 +394,86 @@ export default defineComponent({
...
+          columnName: '#M_Warehouse_ID',
+          isForceSession: true
+        })
+      })
+      store.commit('setProductSearchFieldQueryFilterByAttribute', {
+        containerUuid: uuidForm,
+        attributeKey: 'price_list_version_id',
+        value: getContext({
+          parentUuid: uuidForm,
+          columnName: '#M_PriceList_ID',
```
Comment (EdwinBetanc0urt) [inline src/components/ADempiere/Form/ProductSearch/index.vue]: Please validate if context contains this value to mutate vuex state.

**Review by EdwinBetanc0urt — APPROVED** [review]

File: src/components/ADempiere/Form/ProductSearch/index.vue (hunk 1 above)
Comment (elsiosanchez) [reply src/components/ADempiere/Form/ProductSearch/index.vue]: Resolved in the following commit https://github.com/solop-develop/frontend-core/pull/2213/commits/da5650132e42f856e8def93a45a3cb4248112bf0

File: src/components/ADempiere/Form/ProductSearch/index.vue (hunk 2 above)
Comment (elsiosanchez) [reply src/components/ADempiere/Form/ProductSearch/index.vue]: Resolved in the following commit https://github.com/solop-develop/frontend-core/pull/2213/commits/da5650132e42f856e8def93a45a3cb4248112bf0

**Review by EdwinBetanc0urt — APPROVED** [review]

**Pull request merged**